cctk/orca_file.py

```diff
@@ -126,7 +114,7 @@ def read_file(cls, filename):
                 elif line.strip().startswith("***        THE OPTIMIZATION HAS CONVERGED     ***"):      #### geometry converged
                     successful_opt += 1
                 elif line.startswith("Maximum memory used throughout the entire SCFHESS-calculation:"): #### a frequency job was completed
-                    successful_freq += 1
+                    successful_freq += 1                                                                #### this approach has a liability, it will return a successful frequency job even if the freq job was only the initial hessian used for the TS search
```

Comment:
I think it's OK to report the initial frequency job for a TS job in the count.  That's actually what happens and you can deal with that with logic on the job type.